Popular_Models_Guide/Llama2/trtllm_guide.md

```diff
@@ -26,11 +26,11 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 -->
 
+TensorRT-LLM is Nvidia's recommended solution of running Large Language Models(LLMs) on Nvidia GPUs. Read more about TensoRT-LLM [here](https://github.com/NVIDIA/TensorRT-LLM) and Triton's TensorRTLLM Backend [here](https://github.com/triton-inference-server/tensorrtllm_backend).
```

Comment:
I don't like it very much because it makes the generated markdown look bad... 
eg:
![image](https://github.com/triton-inference-server/tutorials/assets/80359429/789b3dc8-92a8-4271-ae67-ed7ebe2c7ffa)
compared to:
![image](https://github.com/triton-inference-server/tutorials/assets/80359429/c04c122d-888f-4e95-aa51-9a36aed571c7)

Reply:
However google's [markdown guide](https://google.github.io/styleguide/docguide/style.html#character-line-limit) claims otherwise so I digress.

Reply:
Sorry, I should have been clearer. You should only have to drop a single line. The render of the paragraph in the md file should not change. Seems here you might have added two and caused a page break.